draft-ietf-quic-http.md

```diff
@@ -535,72 +526,46 @@ The PRIORITY frame payload has the following fields:
     {{prioritized-element-types}}). When sent on a request stream, this MUST be
     set to `11`.  When sent on the control stream, this MUST NOT be set to `11`.
 
-  DT (Element Dependency Type):
-  : A two-bit field indicating the type of element being depended on (see
-    {{element-dependency-types}}).
-
+  
   Empty:
-  : A four-bit field which MUST be zero when sent and MUST be ignored
+  : A six-bit field which MUST be zero when sent and MUST be ignored
     on receipt.
 
   Prioritized Element ID:
   : A variable-length integer that identifies the element being prioritized.
     Depending on the value of Prioritized Type, this contains the Stream ID of a
-    request stream, the Push ID of a promised resource, a Placeholder ID of a
-    placeholder, or is absent.
-
-  Element Dependency ID:
-  : A variable-length integer that identifies the element on which a dependency
-    is being expressed. Depending on the value of Dependency Type, this contains
-    the Stream ID of a request stream, the Push ID of a promised resource, the
-    Placeholder ID of a placeholder, or is absent.  For details of
-    dependencies, see {{priority}} and {{!RFC7540}}, Section 5.3.
-
-  Weight:
-  : An unsigned 8-bit integer representing a priority weight for the prioritized
-    element (see {{!RFC7540}}, Section 5.3). Add one to the value to obtain a
-    weight between 1 and 256.
-
-The values for the Prioritized Element Type ({{prioritized-element-types}}) and
-Element Dependency Type ({{element-dependency-types}}) imply the interpretation
-of the associated Element ID fields.
+    request stream, the Push ID of a promised resource, or is absent.
+
+  Priority:
+  : An unsigned variable-length integer representing a priority for the prioritized
+    element (see {{!RFC7540}}, Section 5.3). It should be interpreted as a fixed point
+    number with the radix after the first byte. For lengths greater than one, bytes after
+    the first byte should be interpreted as coming after the radix point and decreasing
+    in signifigance, such that 2.5 could be encoded as 0x0280, 1.25 could be encoded as
+    0x0140 and 2.000976562 could be encoded as 0x02040. For simple translation into
+    comparable 64 bit integers, an implementation could interpreted the priorities 
+    internally as the integer value left shifted by 64 minus the length of the field in
```

Comment:
This last definition I completely get and is simple.

Reply:
I think an example would be very helpful here.

Reply:
I added the same examples from above interpreted back from encoded values as examples